style.css

```diff
@@ -12,34 +12,60 @@
     margin-bottom: 0.5em;
 }
 
+
+#batchedit div#summary-top {
+    position: relative;
+    padding: 0.25em;
+    height: 6em;
+}
+
 #batchedit div.file {
+    position: relative;
```

Comment:
Shouldn't div.file-head be relative instead?

Reply:
Probably.